api/src/main/java/org/openmrs/scheduler/TaskFactory.java

```diff
@@ -65,10 +65,8 @@ public Task createInstance(TaskDefinition taskDefinition) throws SchedulerExcept
 			throw new SchedulerException("could not load class", cnfe);
 		}
 		catch (Exception e) {
-			if (log.isDebugEnabled()) {
 				// don't need to log errors here necessarily.  If its needed, the calling method can log it.
-				log.debug("Error creating new task for class " + taskDefinition.getTaskClass(), e);
-			}
+				log.debug("Error creating new task for class {}", taskDefinition.getTaskClass(), e);
```

Comment:
Aren't you one tab too many here?
![image](https://user-images.githubusercontent.com/9057882/92987732-88dad980-f4c5-11ea-850a-a50c7bc99a35.png)